QuasarHelios2015/src/org/team1540/quasarhelios/ControlInterface.java

```diff
@@ -48,9 +49,9 @@ private static void setupRollers() {
             Rollers.direction.set(false);
         });
 
-        povDown.send(() -> {
-            if (!Rollers.direction.get()) {
-                Rollers.running.set(true);
+        povUp.send(() -> {
+            if (!Rollers.direction.get() && Rollers.running.get()) {
+                Rollers.running.set(false);
```

Comment:
When the rollers are going one direction, pressing the button to go the other direction makes more sense to stop the rollers rather than go the other direction.
Both Anna and I found this to be more intuitive.